SmartDeviceLink/SDLStreamingAudioLifecycleManager.m

```diff
@@ -195,6 +196,20 @@ - (void)sdl_handleAudioStartServiceAck:(SDLProtocolMessage *)audioStartServiceAc
     [self.audioStreamStateMachine transitionToState:SDLAudioStreamManagerStateReady];
 }
 
+- (void)handleProtocolEndService:(SDLProtocolMessage *)endService forSession:(Byte)session {
+    // Respond with EndService ACK to address SYNC bug
+    SDLProtocol *sdlProtocol = [[SDLProtocol alloc] init];
+    SDLProtocolHeader *header = [SDLProtocolHeader headerForVersion:(UInt8)[SDLGlobals sharedGlobals].protocolVersion.major];
+    header.frameType = SDLFrameTypeControl;
+    header.serviceType = SDLServiceTypeControl;
+    header.frameData = SDLFrameInfoEndServiceACK;
+    header.sessionID = session;
+    SDLProtocolMessage *message = [SDLProtocolMessage messageWithHeader:header andPayload:nil];
+    [sdlProtocol sdl_sendDataToTransport:message.data onService:header.serviceType];
```

Comment:
Move this logic to the `SDLProtocol` class to a method (maybe call it `endSessionAckWithType`). That way we only need to create the message once and access it from both the `SDLStreamingAudioLifecycleManager` and the `SDLStreamingVideoLifecycleManager`. For reference, look at how we call `endServiceWithType` and `startServiceWithType` in the `SDLStreamingAudioLifecycleManager` and `SDLStreamingVideoLifecycleManager`.

Reply:
Fixed in most recent commit.